virtualcluster/config/setup/all_in_one_capi.yaml

```diff
@@ -235,9 +240,10 @@ spec:
         - manager
         args:
         - --disable-stacktrace=true
-        - --enable-webhook=true
+        - --enable-webhook=false
+        - --master-prov=capi
         image: virtualcluster/manager-amd64
-        imagePullPolicy: Always
+        imagePullPolicy: Never
```

Comment:
This will be handled properly with #130 using `kustomize` to set them differently based on the env/stage, for now `Never` allows us to build images locally and load them into kind for testing.